constellations/gemini-admin/src/classes/classEdit.js

```diff
@@ -102,6 +106,14 @@ export class ClassEditPage extends React.Component {
         this.setState({ [value]: event.target.value });
     };
 
+    onChangeFullState = (e) => {
+        const value = e.target.value;
+        value = parseInt(value);
+        this.setState({
+            fullState: value,
```

Comment:
Close! But looks like it didn't work because of this error which I found in the Console tab of Inspect Element. Do you need a refresher on Inspect? We scratched the surface in the web dev class, but I could've been more detailed with it.

![Screen Shot 2020-07-15 at 1 19 41 AM](https://user-images.githubusercontent.com/4695528/87506515-7b29e480-c639-11ea-9b5a-6bb833a907b9.png)

Looks like `value` is a constant so it can't be changed. You'll have to use a new variable or simply do this:
```
this.setState({
    fullState: parseInt(value)
});
```